### PR TITLE
[MIG] [17.0] hr_applicant: migrated module.

### PR DESCRIPTION
**Pull request by ShwetaSerpentCS**
Responsible : Utsav

### CODE REVIEW

File: hr_applicant/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+{
+    "name": "Applicant Process",
+    "version": "17.0.0.1.0",
+    "category": "Human Resources",
+    "sequence": 90,
+    "license": "LGPL-3",
+    "summary": "Applicant and Employee Subsections, Training",
+    "author": "Serpent Consulting Services Pvt. Ltd.",
+    "website": "http://www.serpentcs.com",
```
Comment (Trivedi-Vacha-SerpentCS) [inline hr_applicant/__manifest__.py]:
```suggestion
    "website": "https://www.serpentcs.com",
```

File: hr_applicant/models/training.py [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+
+class TrainingClass(models.Model):
+
+    _name = "training.class"
+    _rec_name = "course_id"
+    _description = "Training Class"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+
+    # @api.constrains("training_start_date", "training_end_date")
+    # def _check_training_dup(self):
```
Comment (Trivedi-Vacha-SerpentCS) [inline hr_applicant/models/training.py]: Remove commented code

File: hr_applicant/models/training.py [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+        string="Course Type",
+        readonly=True,
+    )
+    training_location = fields.Char(
+        related="course_id.training_location", string="Training Location", readonly=True
+    )
+    training_attendees = fields.Integer("Training Attendees", required=True)
+    training_start_date = fields.Date(
+        "Training Start Date",
+        # readonly=True,
```
Comment (Trivedi-Vacha-SerpentCS) [inline hr_applicant/models/training.py]: if this is managed from xml side then remove it from py

File: hr_applicant/wizard/select_training.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    _name = "select.training"
+    _description = "Select Training"
+
+    is_training_needed = fields.Boolean(string="Is Training needed?", required=True)
+    applicant_id = fields.Many2one('hr.applicant', "Applicant", store=True)
+    job_id = fields.Many2one(related="applicant_id.job_id", store=True)
+    training_courses_ids = fields.Many2many("training.class", string="Training")
+
+    # @api.onchange("is_training_needed")
+    # def _onchange_training_courses(self):
```
Comment (Trivedi-Vacha-SerpentCS) [inline hr_applicant/wizard/select_training.py]: Remove commented code if not require for future use

File: hr_applicant/security/ir.model.access.csv [hunk 5]
```diff
@@ -0,0 +1,18 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_applicant_education,applicant.education,model_applicant_education,base.group_system,1,1,1,1
```
Comment (Trivedi-Vacha-SerpentCS) [inline hr_applicant/security/ir.model.access.csv:2]: every user having 1111 access??
We should manage the access, like user cannot delete 1110 and office can have delete access 1111
@ShwetaSerpentCS

File: hr_applicant/models/training.py [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+        "Employee",
+        # readonly=True,
+        # states={"draft": [("readonly", False)]},
+    )
+    attendees_image = fields.Binary(related="employee_id.image_1920", string="Image")
+    training_start_date = fields.Date(
+        "Training Start Date",
+        required=True,
+        # readonly=True,
+        # states={"draft": [("readonly", False)]},
```
Comment (Trivedi-Vacha-SerpentCS) [inline hr_applicant/models/training.py:224]: remove commented code

**Review by Trivedi-Vacha-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by ketanserpentcs** [comment]
Testing is assigned to Pooja

**Pull request merged**